src/extension/module.js

```diff
@@ -3941,6 +3948,14 @@ import sampleRUM from './rum.js';
         customViewUrl.searchParams.set('path', status.webPath);
         envUrl = customViewUrl;
       }
+
+      const liveDomains = ['aem.live', 'hlx.live'];
+      if (cacheBust
```

Comment:
This `if` statement hurt to write.. Open to suggestions for improvement..

Reply:
- Looking at the code above, `envUrl` can either be a `string` or a `URL`, which can mess things up. I'd make sure it's always a `URL` which makes it cleaner to manipulate
- Why are you excluding `prod` and `liveDomains`? I'd assume cache busting will be necessary there as well

Reply:
> Why are you excluding prod and liveDomains
Yeah, that's why I said it hurt.. It's not excluding live but I can 100% see why it's hard to reason about what this if statement is checking for.

Basically it's 
IF cacheBusting was passed AND NOT (targetEnv=prod and not going to `hlx.live` or `aem.live` and in transient mode)

This statement makes sure we don't include the cacheBuster on a prod domain when in transient mode.

Check slack when you are back.. I recording a demo on why this is needed.